components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -113,6 +115,7 @@ export const Default = Template.bind({});
 Default.args = {};
 
 export const WithLabel = Template.bind({});
+WithLabel.storyName = "With label";
```

Comment:
Do we need the `!dev` tag here, too?

Reply:
I had left it how it was, but I think that is a good idea. I've added that tag to remove it from the sidebar in the latest update.